bin/circleci/setup-integration-tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -ex
+
+# TODO: Update this occasionally from here:
```

Comment:
Might be worth doing that... I didn't want to automate it, so we could maybe pin it to a known version. Might also be interesting someday to expand this into a set of versions to try against, because at some point we'll be spread across different versions that yield different onboarding experiences (e.g. when we start showing up in add-on install whitelist)